source/CardGames.py

```diff
@@ -156,6 +156,16 @@ def highest_card(self):
         result_card = card
     return result_card
 
+  def calculateHand(self):
+    total = self.showValue()
+    for card in self.hand:
+      if(total < 21):
+        break
+      if(card.value == 11): # Ace
+        total -= 10
+
+    return total
```

Comment:
Great implementation of hand calculation. The Ace value change feature works well.